### DISCUSSION TITLE
Add support for mixing WhatsApp and Telegram

### DISCUSSION THREAD

**Discussion by martin-braun** [d99kris/nchat#111] [Ideas]
It seems I can only setup one thing at a time. I can use different config dirs though, but it would be nicer to have it all in one.

**Comment by d99kris** [comment]
Hi @martin-braun - running both in one instance of nchat is not (officially) supported at this point. See #102 for more details.

The question has come up before though, so I might look into adding official support for it eventually, but not in a near future.

**Reply by d99kris** [reply]
Hi again @martin-braun - I finally got around to take a look at this. Support for multiple protocols was added now, see #140.

**Reply by martin-braun** [reply]
@d99kris Awesome, thank you for your contribution, I will attempt this in the future and report back.
